markdown/src/viewer.c

```diff
@@ -389,8 +389,8 @@ markdown_viewer_update_view(MarkdownViewer *self)
      * position once the webview is reloaded. */
     if (self->priv->load_handle == 0) {
       self->priv->load_handle =
-        g_signal_connect_swapped(WEBKIT_WEB_VIEW(self), "notify::load-status",
-          G_CALLBACK(on_webview_load_status_notify), self);
+        g_signal_connect_swapped(WEBKIT_WEB_VIEW(self), "notify::is-loading",
+          G_CALLBACK(on_webview_is_loading_notify), self);
```

Comment:
Instead of the property notification, maybe it should use [`load-changed`](https://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebView.html#WebKitWebView-load-changed)?